demo/chbench/mzcompose.yml

```diff
@@ -281,6 +281,7 @@ volumes:
 
   # shared volumes for load tests, but a mounted directories for benchmarks
```

Comment:
Now that there are multiple env vars that affect this, could you maybe document all of them? I'm not sure where the best place to do it is, but here seems fine.

Reply:
Sure, I can document this. The main environment variable is meant be MZ_CHBENCH_SNAPSHOT and the KAFKA/ZK ones are only meant to be internal to the mzcompose file.